packages/server/graphql/private/mutations/runScheduledJobs.ts

```diff
@@ -36,23 +36,17 @@ const processMeetingStageTimeLimits = async (
   })
 }
 
-export type ScheduledJobUnion = ScheduledJobMeetingStageTimeLimit | ScheduledJobOrganizationLock
+export type ScheduledJobUnion = ScheduledJobMeetingStageTimeLimit | ScheduledTeamLimitsJob
 
 const processJob = async (job: ScheduledJobUnion, dataLoader: DataLoaderWorker) => {
   const r = await getRethink()
   const res = await r.table('ScheduledJob').get(job.id).delete().run()
   // prevent duplicates. after this point, we assume the job finishes to completion (ignores server crashes, etc.)
   if (res.deleted !== 1) return
-
-  if (job.type === 'MEETING_STAGE_TIME_LIMIT_END') {
-    return processMeetingStageTimeLimits(
-      job as ScheduledJobMeetingStageTimeLimit,
-      dataLoader
-    ).catch(console.log)
-  } else if (job.type === 'LOCK_ORGANIZATION') {
-    return processLockOrganizationJob(job as ScheduledJobOrganizationLock, dataLoader).catch(
-      console.log
-    )
+  if ('orgId' in job) {
```

Comment:
I was checking a `type` here because in future, we may have more job types added and they may also have `orgId`. What do you think?

Reply:
Are there any jobs you have in mind that will be added soon? If so, I'm happy to cast the type, but if not, I think it's best to check for the `orgId`

Reply:
Make sense, no other jobs for now, could be kept as is and changed later if needed